docs/commands/console.rst

```diff
@@ -0,0 +1,6 @@
+console
```

Comment:
Due to a `sphinx-click` bug, we have to manually add titles for `console` and `run`